src/state_machine.zig

```diff
@@ -2504,8 +2905,58 @@ test "create/lookup 2-phase transfers" {
         \\ commit create_transfers
 
         // Check balances after resolving.
-        \\ lookup_account A1 15 36  0  0
-        \\ lookup_account A2  0  0 15 36
+        \\ lookup_account A1  0 36  0  0
+        \\ lookup_account A2  0  0  0 36
+        \\ commit lookup_accounts
+    );
+}
+
+test "create/lookup expired transfers" {
+    try check(
+        \\ account A1  0  0  0  0  _  _  _ _ L1 C1   _   _   _ _ _ _ ok
+        \\ account A2  0  0  0  0  _  _  _ _ L1 C1   _   _   _ _ _ _ ok
+        \\ commit create_accounts
+
+        // First phase.
+        \\ transfer   T1 A1 A2   10   _  _  _  _    0 L1 C1   _ PEN   _   _   _   _  _ _ ok // Timeout zero will never expire.
+        \\ transfer   T2 A1 A2   11   _  _  _  _    1 L1 C1   _ PEN   _   _   _   _  _ _ ok
+        \\ transfer   T3 A1 A2   12   _  _  _  _    2 L1 C1   _ PEN   _   _   _   _  _ _ ok
+        \\ transfer   T4 A1 A2   13   _  _  _  _    3 L1 C1   _ PEN   _   _   _   _  _ _ ok
+        \\ commit create_transfers
+
+        // Check balances before expiration.
+        \\ lookup_account A1 46  0  0  0
+        \\ lookup_account A2  0  0 46  0
+        \\ commit lookup_accounts
+
+        // Check balances after 1s.
+        \\ tick 1000000000
```

Comment:
Nit, but I'd suggest modifying `tick` to take a unit enum, so you can instead write `tick 1 seconds`